csharp/samples/getting-started-receiving.md

```diff
@@ -0,0 +1,201 @@
+# Get started with Event Hubs in .NET
```

Comment:
>Get started with Event Hubs in .NET [](start = 2, length = 35)

Shouldn't this be covering low level API in 3-4 lines of code instead of full blown EPH? We can dedicate another readme for EPH. But for a starter sending single event and receiving it back should be the starting point.